### DISCUSSION TITLE
Enteric coated formulation

### DISCUSSION THREAD

**Discussion by Ricardo-DO** [Open-Systems-Pharmacology/Forum#1186] [Questions & Problems] [answered]
Dear all,

I hope you are having a good time.
I would like to ask you if you have a workaround for an administration of an enteric coated formulation using a mechanistic approach.
The solubility at low pH is much higher than at pH 6, so the if the drug is released in the intestine, the amount dissolved will be much lower (~10% of the IR formulation).

I have attempted this without success

1. Tried with an advanced protocol setting the administration in the Small intestine mucosa, but it only accepts Table, Zero or First order and Weibull, not Particle Dissolution.
2.  Tried using a plain oral administration and setting the stomach solubility at 0 but is dissolving there anyway.
3. Tried to set the stomach transit time at the lowest, but it seems that 1 min is enough to dissolve the formulation.

Have you got any suggestions to do this in PKSim or MoBi? I mean using the Particle dissolution method in the Duodenum.

Many thanks,
Ricardo

**Comment by AndreDlm** [comment, marked as answer]
Dear Ricardo,

unfortunately, there is no simple solution for this, but the following workaround in MoBi actives particle dissolution once the tablet has entered the duodenum: 

1) In your simulation, add a second oral administration that occurs at the same time than the enteric coated tablet. Set the dose of this administration to 0 mg and the water volume ingested with the dose to 0 mg/kg:
![image](https://user-images.githubusercontent.com/26541603/165071813-be27b899-27ee-4c10-abaa-1f286740a6d4.png)
As formulation for this dummy administration you can use e.g. the table input with arbitrary values, but not `Dissolved` - you need a solid oral dosage form as a dummy to track the location of your administered formulations in the GI tract. 

2) Send your simulation from PK-Sim to MoBi, navigate to the administration of the particle formulation in the `Events` building block and set the `Number of particles fraction Stomach` to 0 and the `Number of particles fraction Duodenum` to 1:
![image](https://user-images.githubusercontent.com/26541603/165072364-4296d602-dfea-436e-b127-6929cc33cd56.png)

3) Now you need to change the start event triggering the dissolution of the particle formulation under `...|Application_StartEvent` for each particle size bin. The default setting here triggers dissolution with the start time of the simulation, but you need to add a new formula that uses the location of the dummy tablet (i.e. the entering of the dummy tablet in the duodenum) as start event for the dissolution of the particle formulation:
![image](https://user-images.githubusercontent.com/26541603/165074004-c98cb387-4be7-4e48-8f4d-5571b7c36fbb.png)

4) You may optionally make some parameters plottable, e.g. to track the location of the dummy tablet and to visualize the number of particles fraction in the GI segments:
![image](https://user-images.githubusercontent.com/26541603/165074895-d4f1e3a9-5885-490c-9aaa-941ebe08e808.png)

5) Update your simulation using the changed `Events` building block; you may also want to send back your simulation to PK-Sim now (optionally). 

When simulating now, the `Number of particles fraction Stomach` should be constantly 0, while the `Number of particles fraction Duodenum` is 1 as long as the dissolution of the particle formulation is inactive. As soon as the dummy tablet enters the duodenum (i.e. the parameter `Tablet_Location` increases to 3), the dissolution of the particle formulation starts, as indicated by the change in the  `Number of particles fraction {Small_Intestinal_Segments}`:
![image](https://user-images.githubusercontent.com/26541603/165076396-a5422535-2afe-4de2-abde-7ee04f7778ed.png)

A MoBi file where the suggested workflow is implemented is attached here:
[EntericCoatedTablet.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/8554043/EntericCoatedTablet.zip)


Note however, that particle dissolution in this workaround is triggered by tablet location and **not** by the pH value in the lumen. 

I hope this helps!

André

**Reply by Ricardo-DO** [reply]
Dear André,

Many thanks for your detailed answer!
Really appreaciate that you have taken the time to respond.
I have tried it and it works beautifully.

Best, 
Ricardo

**Comment by Ricardo-DO** [comment]
Dear Community,

I hope you are doing well.
Following André procedure I get a good approximation for an enteric coated formulation.
I have noticed that even changing the Gastric Transit Time, the dummy tablet reaches duodenum after 1h.

Do you know which parameter I can change in PKSim or in MoBi, to make the tablet reach duodenum in a shorter time? e.g. 0.5 h

Many thanks,
Ricardo

**Reply by Yuri05** [reply]
I guess you are looking for this:
![grafik](https://user-images.githubusercontent.com/25061876/188153756-9db7670b-ec35-44d6-9bec-fff016c22dab.png)

**Reply by Ricardo-DO** [reply]
Hello Yuri,

That's what  I was lloking for!

Many thanks,
Ricardo

**Comment by Ricardo-DO** [comment]
Dear community,

I was trying to use the same approach but for a Weibull function for dissolution.
Unfortunately, I had no success.

I was setting the release rate fraction in Duodenum to 1 and 0 in Stomach, trying to do something similar to the number of particles.
Do you know if this can be done and what would be the steps, or this is something that it just can't be done.

I was using a Weibull with tlag in PKSim, but if I want to use it in fed state, then it would not work.

Many thanks,
Ricardo

**Reply by Yuri05** [reply]
>I was setting the release rate fraction in Duodenum to 1 and 0 in Stomach, trying to do something similar to the number of particles.

Weibull and comparable formulations provide 2 alternative dissolution modi: if `Use as suspension` is selected, the formulation will disintegrate in the stomach and the disintegrated particles will migrate along the gastrointestinal tract compartments (_particle dissolution_ formulation is always treated as a suspension per construction). If, on the other side, this option is not selected, the tablet will be treated as nondisintegrating tablet with discrete transition in the different intestinal compartments. 

![grafik](https://user-images.githubusercontent.com/25061876/190629791-96c3060e-dda1-40cb-896f-acfe13bc4ee3.png)

Now depending on the selected use case, **different model parameters are used**:

![grafik](https://user-images.githubusercontent.com/25061876/190632149-c9e5fea3-12ae-45a7-baf1-acf500f87b23.png)

So if you have a Weibull formulation defined as **suspension** (which is the default state in PK-Sim): you must modify "`Suspension fraction XXX`" parameters. ("`Release rate XXX`" parameters are not used at all in this case and modifying them will have no effect on the model)

**Comment by Ricardo-DO** [comment]
Hi Juri,

Many thanks for your reply.
I am still having problems with the Weibull and the Table approach.
I have 2 molecules with high solubility in pH 6 (9 and 50 mg/mL, respectively).
With the same Weibull function for both molecules, the one with 50 mg/mL at pH 6 was set in the Duodenum and the other was still in the stomach.

The one in the duodenum is just not dissolving, the one in the stomach doesn't have any problem.

This was the setup for the high soluble compound in duodenum with Weibull as a suspension
![image](https://user-images.githubusercontent.com/59565610/190987307-4ea294d5-9478-457c-97f0-bf22218ab2b5.png)

These are the results
![image](https://user-images.githubusercontent.com/59565610/190987568-739da511-6af1-42e6-b816-7210398a6f13.png)

While the one in stomach dissolves quickly to 100%, the one in duodenum dissolves only up to 5%,

Also, if I leave both formulations in the stomach, the plasma concentration is a lot higher than if it is set in the duodenum.
![image](https://user-images.githubusercontent.com/59565610/190990872-dc7723ee-98ed-4a55-811b-f671e8c7e147.png)

And this is with the formulation in duodenum
![image](https://user-images.githubusercontent.com/59565610/190990961-a1ab9739-dfb9-48f3-8b96-44750eebcfbf.png)

Using table dissolution, I just don't get anything dissolved of the compound in duodenum, the one in the stomach works well.
![image](https://user-images.githubusercontent.com/59565610/190991172-4db6e535-308f-40ee-b7fd-a6e245b58167.png)

Have you got any suggestion on how to implement the Weibull or table dissolution in duodenum?
I was following the same steps as for Particle dissolution, but with Weibull and Table approaches with the modifications suggested by Juri.

Many thanks,
Ricardo

**Reply by Yuri05** [reply]
I guess what is missing is the adjustment of the application start time parameter for the "duodenum" tablet.
Because in case of Weibull (unlike particles dissolution) the start time of the application is explicitly used in the dissolution equation, it must be se to the current time when the tablet reaches duodenum:
![grafik](https://user-images.githubusercontent.com/25061876/191600874-8a751f9f-e006-4eaa-bc30-20f3b59f092b.png)

**Comment by Ricardo-DO** [comment]
Hi Juri,

Many thanks for your help.
Now both formulations have the same dissolution profile but one in stomach and the other in duodenum.

I'll leave this screenshot of how to Set up the assignment, for if someone has the same query in the future

![image](https://user-images.githubusercontent.com/59565610/191952366-9f62ef8d-bf13-4fc5-93a0-02a4427e6395.png)


Best,
Ricardo

**Comment by Ricardo-DO** [comment]
Dear all,

I hope you are well.
I was trying to simulate an administration of a formulation targeting colon release. I was using the PSD approach.
While the particles are in colon and then move forwards, absorption starts in duodenum-> jejunum...
Is there another setting that needs to be adjusted to start the absorption in colon?

<img width="1065" height="807" alt="PKSim" src="https://github.com/user-attachments/assets/8f453fe9-f197-4748-bb21-32ddc5a7e292" />

Many thanks,
Ricardo

**Reply by RobinM92** [reply]
Hi @Ricardo-DO,

I would recommend you to start a new issue for this question as this one has been marked 'answered' so it might not pop up. 
Would also be interested in the answer, but AFAIK the PSD approach only works for IR formulations (so far)
